crates/rpc-types/src/eth/block.rs

```diff
@@ -75,11 +75,11 @@ pub struct Header {
     /// Difficulty
     pub difficulty: U256,
     /// Block number
-    pub number: Option<U256>,
+    pub number: Option<u64>,
     /// Gas Limit
-    pub gas_limit: U256,
+    pub gas_limit: u128,
     /// Gas Used
-    pub gas_used: U256,
+    pub gas_used: u128,
     /// Timestamp
     pub timestamp: U256,
```

Comment:
maybe followup work, but should `timestamp` be a `u64` cc @mattsse?

Reply:
`timestamp` is `u64` in certain places. I'm in favour

Reply:
sg!